contracts/contracts/harvest/OETHHarvesterSimple.sol

```diff
@@ -5,42 +5,83 @@ import { Strategizable } from "../governance/Strategizable.sol";
 import { IStrategy } from "../interfaces/IStrategy.sol";
 import { IERC20 } from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
 import { SafeERC20 } from "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+import { Initializable } from "../utils/Initializable.sol";
 
-contract OETHHarvesterSimple is Strategizable {
+/// @title OETH Harvester Simple Contract
+/// @notice Contract to harvest rewards from strategies
+/// @author Origin Protocol Inc
+contract OETHHarvesterSimple is Initializable, Strategizable {
     using SafeERC20 for IERC20;
 
+    ////////////////////////////////////////////////////
+    /// --- CONSTANTS & IMMUTABLES
+    ////////////////////////////////////////////////////
+    /// @notice WETH address
+    address public immutable WETH;
+
     ////////////////////////////////////////////////////
     /// --- STORAGE
     ////////////////////////////////////////////////////
+    /// @notice Gap for upgrade safety
+    uint256[50] private ___gap;
```

Comment:
Usually we place the __gap after the local properties (after `supportedStrategies` mapping) and make it so that local variables + the gap make up a round number. In this case since the `dripper` takes 1 slot and `supportedStrategies` another one, `uint256[48] private ___gap; ` would be better.

Reply:
I thought we were using gaps to prevent reusing slots that have already been allocated in the previous implementation. That’s why I placed it at the front.

In the PR description, I included a picture showing that `7` slots are already in use. Therefore, I added a gap before introducing new storage variables to avoid any conflicts.
I’m not an expert on proxies and may not be as familiar with this as some of you. Please let me know if this doesn’t make sense!

If it does, should I consider using `41` or `43` for the gap to reach a round number?

Reply:
Yes, its important to line up the slots so they are backward compatible with the exiting `OETHHarvester`

Reply:
ok, after talking through this one. Let's not upgrade the exiting `OETHHarvesterProxy`. Instead we'll do what the current scripts do which is deploy and new `OETHHarvesterSimpleProxy` and `OETHHarvesterSimple` implementation. All the strategies can then be changed to use the new harvester

Reply:
Sorry for the confusion. 
I’ve changed the gap position and value accordingly to @sparrowDom comment: https://github.com/OriginProtocol/origin-dollar/pull/2363/commits/5e7f623eed8b3472cebceb5841a64f110204b252.